docs/setup/schema.md

```diff
@@ -150,11 +150,11 @@ Informatics Gateway validates all configuration options at startup. Any provided
 Informatics Gateway, by default, is configured to writes all logs to the console as well as text files. The behaviors may be changed in the `Logging` section of the `appsettings.json` file.
 
 > [!Note]
-> If the Informatics Gateway is running inside a Docker container, additional configuration may be required to limit the size to prevent filling up storage space. Please refer to [Docker](https://docs.docker.com/config/containers/logging/configure/) for additional information.
+> If the Informatics Gateway is running inside a Docker container, additional configuration may be required to limit the size to prevent filling up storage space. Refer to the [Docker documentation](https://docs.docker.com/config/containers/logging/configure/) for additional information.
 
 #### Log Levels
 
-Log level may be adjusted on a per-module basis. For example, given the following log entries:
+The log level may be adjusted on a per-module basis. For example, given the following log entries:
```

Comment:
This language suggests that the paragraph following the example (on line 166) will be related to it. However, that paragraph doesn't seem to discuss the example.

Reply:
How about something like:

By default, the `Monai` namespace is to log all `Information` level logs. However, log level may be adjusted on a per-module basis. For example, given the following log entries:

```
 14:26:13 info: Monai.Deploy.InformaticsGateway.Services.Connectors.WorkloadManagerNotificationService[0]
      MONAI Workload Manager Notification Hosted Service is running.
 14:26:13 info: Monai.Deploy.InformaticsGateway.Services.Storage.SpaceReclaimerService[0]
      Disk Space Reclaimer Hosted Service is running.
```

If additional information is required to debug the **WorkloadManagerNotificationService** module or to turn down the noise, add a new entry under the `LogLevel` section of the configuration file to adjust it:

```
 "Logging": {
    "LogLevel": {
      "Monai": "Information",
      "Monai.Deploy.InformaticsGateway.Services.Connectors.WorkloadManagerNotificationService": "Debug",
      ...
```